### PR TITLE
Content changes to stocks form types 0009 and 0010.

### PR DESCRIPTION
**Pull request by samiwel**
### What is the context of this PR?
A change request was made for stocks form types 0009 and 0010.
This PR makes small changes to the content as requested.

### How to review 
Ensure that both schemas can be launched and content is as described on Trello cards.

### CODE REVIEW

File: data/en/stocks_0010.json [hunk 1]
```diff
@@ -123,8 +123,8 @@
                                         {
                                             "list": [
                                                 "Products that you own in intermediate stages of completion (even if not held by you)",
-                                                "Long term business contract balances in line with UK Generally Accepted Accountancy Practice or International GAAP",
-                                                "All stocks owned; whether held by you or in transit in the UK or abroad",
+                                                "Long term business contract balances in line with UK Generally Accepted Accountancy Practice (GAAP) or International GAAP",
```
Comment (tgandrews) [inline data/en/stocks_0010.json:126]: There is no definition of GAAP but it is used. I'm no writing expert but this looks weird

**Review by tgandrews — APPROVED** [review]
Haven't tested it fully but the surveys launch and the changes look fine.

File: data/en/stocks_0010.json (hunk 1 above)
Comment (samiwel) [reply data/en/stocks_0010.json:126]: Generally Accepted Accountancy Practice?

Comment (tgandrews) [reply data/en/stocks_0010.json:126]: Yeah I know it is but this change removes the definition. That's what I think is weird.

I only know what GAAP means by either: guessing from the sentence context or, by knowing it up front.

Comment (tgandrews) [reply data/en/stocks_0010.json:126]: You can ignore this as this is content. I just think its odd

Comment (samiwel) [reply data/en/stocks_0010.json:126]: Doesn't this change add the definition?

**Review by rmccar — APPROVED** [review]

**Review by SamGodwin2 — APPROVED** [review]

**Pull request merged**